test/extensions/filters/http/compressor/compressor_filter_test.cc

```diff
@@ -812,6 +812,135 @@ TEST_F(MultipleFiltersTest, UseFirstRegisteredFilterWhenWildcard) {
   EXPECT_EQ(1, stats2_.counter("test2.compressor.test2.test.header_wildcard").value());
 }
 
+class ChooseFirstTest
+    : public MultipleFiltersTest,
```

Comment:
Is there any chance we can share some code between MultipleFiltersTest or CompressorFilterTest? there are a lot of method duplicated with CompressorFilterTest. But it also seems ok to test those case like MultipleFiltersTest way, only ensure which filter is used.

Reply:
Yeah, I also think so, but it seems a lot of work to rewrite this test. maybe I can have another PR to do this work.

Reply:
Could you add a TODO comment here?

Reply:
OK, done, thanks!